JavaScript/JavaScriptSDK/Context/PageVisitTimeManager.ts

```diff
@@ -0,0 +1,96 @@
+﻿
```

Comment:
Good point. Is there a folder you think that is a good match? I couldnt find one that felt like a good fit - I can also add a new one. Let me know what you prefer...